crypto/onetimesig.go

```diff
@@ -157,12 +157,12 @@ type OneTimeSignatureSecretsPersistent struct {
 	// The odd `codec:` name is for backwards compatibility with previous
 	// stored keys where we failed to give any explicit `codec:` name.
 	FirstBatch uint64            `codec:"First"`
-	Batches    []ephemeralSubkey `codec:"Sub,allocbound=-"`
+	Batches    []ephemeralSubkey `codec:"Sub,allocbound=8888"`
 
 	// FirstOffset denotes the first offset whose subkey appears in Offsets.
 	// These subkeys correspond to batch FirstBatch-1.
 	FirstOffset uint64            `codec:"firstoff"`
-	Offsets     []ephemeralSubkey `codec:"offkeys,allocbound=-"`
+	Offsets     []ephemeralSubkey `codec:"offkeys,allocbound=8888"`
```

Comment:
@tsachiherman do you think these need to be bounded? Any idea what will be a good bound?

Reply:
I will move this bound to a separate task/PR

Reply:
moved to https://github.com/algorand/go-algorand/pull/3196